hw2/05-form.js

```diff
@@ -1 +1,29 @@
-// Add your code here
+//Grab the form element.
+const form = document.getElementById('hwForm')
+//Event listener for the form submit event.
+form.addEventListener('submit', function (event) {
+    event.preventDefault()
+    const name = event.target.elements.name.value //This field, email and menu are required, so we can just grab the value.
+    const email = event.target.elements.email.value
+    const menu = event.target.elements.regMenu.value
+    //Make an array of checkboxes, filter for checked boxes, map to get the value.
+    const courses = Array.from(event.target.elements.courses)
+        .filter((checkbox) => checkbox.checked)
+        .map((checkbox) => checkbox.value)
+    //If no courses are selected, set the courses variable to a string.
+    if (courses.length < 1) {
+        courses = `None selected!`
```

Comment:
These are cases to use let instead of const.
<img width="609" alt="Screenshot 2023-04-25 at 2 22 55 AM" src="https://user-images.githubusercontent.com/66976595/234233513-023c21f8-54d0-46a6-ad66-35d3b58322b3.png">